src/library/helpers/standards-helpers.js

```diff
@@ -211,14 +212,17 @@ var NgssHelper = function() {
                     Object.keys(groupMap).map( function(title) {
                         var statements = groupMap[title];
                         return [
-                            title,
+                            b({}, title),
                             br(),
                             statements.map( function(s) {
                                 var arrDesc = s.description;
                                 if(arrDesc.length < 3) { return null; }
                                 var desc = "";
                                 for(var i = 2; i < arrDesc.length; i++) {
                                     desc += arrDesc[i];
+            				        if(arrDesc[i].endsWith(".")) {
```

Comment:
`endsWith` isn't supported in IE but it is in Edge.  I'm not sure what the current browser support for the portal should be.